src/DataContainer/GoogleMap.php

```diff
@@ -182,7 +182,7 @@ public function checkPermission()
                                 $arrModulep = \StringUtil::deserialize($objGroup->contao_google_maps_bundlep);
 
                                 if (\is_array($arrModulep) && \in_array('create', $arrModulep)) {
-                                    $arrModules = \StringUtil::deserialize($objGroup->contao_google_maps_bundles, true);
```

Comment:
I think the problem here is more that we use the compat layer. Could you try to replace

`\StringUtil::deserialize` by `StringUtil::deserialize` and `use Contao\StringUtil`?

Reply:
I added some not all (because I did not want to get to many changes!) definitions from https://github.com/contao/easy-coding-standard/blob/3.x/config/set/contao.php to find fixes ;-)